qownnotes/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: qownnotes
-version: 16.09.0
+version: 16.09.2
```

Comment:
Mind adding quotes around the version as well? That will always enforce this to be a string and avoid weird results the day you release 16.10 for instance :) (as it will  be 16.1 otherwise)